README.md

```diff
@@ -14,6 +14,9 @@ The script utilises the following environment variables.
 | AWS_REGION            | Yes      | -             | AWS region to connect to                                                    |
 | MIN_SEVERITY          | No       | `ALL`         | Sets the minimum vulnerability severity level to indicate a failure         |
 | MAX_WAIT              | No       | `120`         | Sets how long, in seconds, to wait for ECR scan results to become available |
+| SLACK_DISABLE         | No       | -             | If set and non-empty, will disable sending of Slack reports                 |
```

Comment:
Is `SLACK_DISABLE` needed? Can we not infer this from the presence or absence of `SLACK_CHANNEL` and `SLACK_WEBHOOK`?

Reply:
@Jata2k Updated in [6c8f9b4](https://github.com/companieshouse/ci-ecr-scan-check/pull/2/commits/6c8f9b456a27cb6b89b81140355952e288b92105). Slack reporting is now based on whether `SLACK_CHANNEL` is set or not. This logic is used to determine whether `SLACK_WEBHOOK` is validated or not.